tests/test_packer.py

```diff
@@ -3,28 +3,48 @@
 from packer import Installer
 
 class TestPacker(unittest.TestCase):
+    test_template = './tests/docker.pkr.hcl'
+    packer = Packer(template=test_template)
+
+    @classmethod
+    def setUpClass(self):
+        self.packer.init('tests')
+
     def test_init(self):
+      self.packer.init('tests')
       self.assertEqual("TODO", "FIXME")
```

Comment:
We should check and see if `packer init` creates any files on disk.

Reference(s):
- https://stackoverflow.com/q/59121161/8507637